webapp/javascript/components/FlameGraph/FlameGraphComponent/Tooltip.tsx

```diff
@@ -34,6 +34,8 @@ export type TooltipProps = {
   numTicks: number;
   leftTicks: number;
   rightTicks: number;
+
+  palette?: FlamegraphPalette;
```

Comment:
Why not force a palette to always be passed? That way we don't have to second guess ourselves.

Reply:
That was done in order to keep unit tests as is :-) and for the clarity of code. I'll make it explicit on tests

Reply:
Fixed!